src/epd5in65f/mod.rs

```diff
@@ -229,11 +233,10 @@ where
         self.interface.cmd_with_data(spi, command, data)
     }
 
-    fn wait_busy_high(&mut self) {
-        self.interface.wait_until_idle(true);
-    }
-    fn wait_busy_low(&mut self) {
-        self.interface.wait_until_idle(false);
+    fn wait_busy_low(&mut self, delay: &mut DELAY) {
+        while self.interface.is_busy(false) {
+            delay.delay_ms(5);
```

Comment:
Whats the reasoning for the 5ms here instead of the 10ms we used elsewhere?

Reply:
The reasoning it that I forgot this case :-)